tests/client/amo/components/TestAddonDetail.js

```diff
@@ -441,12 +441,15 @@ describe('AddonDetail', () => {
     });
 
     it('prompts you to read many reviews', () => {
+      const i18n = getFakeI18nInst();
       const root = reviewFooterDOM({
         ratingsCount: 5,
+        i18n,
       });
       const footer =
         root.querySelector('.AddonDetail-read-reviews-footer');
       assert.equal(footer.textContent, 'Read all 5 reviews');
+      assert.ok(i18n.ngettext.calledWith('Read %(count)s review', 'Read all %(count)s reviews', 5));
```

Comment:
Ah, ok - that's already handled here: https://github.com/mozilla/addons-frontend/blob/2754e5c7faf719f65527fc853afb4243876d7df3/tests/client/amo/components/TestAddonDetail.js#L438